scripts/convert_t5_to_longformerencoderdecoder.py

```diff
@@ -0,0 +1,120 @@
+import argparse
+import logging
+import os
+import copy
+
+from transformers import T5Tokenizer, T5ForConditionalGeneration
+from longformer.longformer_encoder_decoder import (
+    LongformerSelfAttentionForT5,
+    LongformerT5Config,
+    LongformerT5ForConditionalGeneration,
+)
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def create_long_model(save_model_to, base_model, attention_window, max_pos):
+    # load base model & tokenizer
+    tokenizer = T5Tokenizer.from_pretrained(base_model, model_max_length=max_pos)
+    model = T5ForConditionalGeneration.from_pretrained(base_model)
+    print("Base model architecture")
+    print(model)
+
+    # setup config
+    config = LongformerT5Config.from_pretrained(base_model)
+    config.architectures = [
+        "LongformerT5ForConditionalGeneration",
+    ]
+    # in T5 attention_probs_dropout_prob is dropout_rate
+    config.attention_probs_dropout_prob = config.dropout_rate
+    config.attention_window = [attention_window] * config.num_hidden_layers
+    config.attention_dilation = [1] * config.num_hidden_layers
```

Comment:
when increasing the model length we probably want to increase number of relative position buckets as well `config.relative_attention_num_buckets`